core/src/main/java/org/infinispan/manager/CacheManagerInfo.java

```diff
@@ -107,6 +108,11 @@ public String getClusterName() {
       return configurationManager.getGlobalConfiguration().transport().clusterName();
    }
 
+   public String getLocalSite() {
+      SiteConfiguration sites = configurationManager.getGlobalConfiguration().sites();
```

Comment:
IMO, you should get the site name from RELAY2. the site-name in global configuration isn't mandatory to set and it may be diffeent.

Reply:
+1, this seems a random arbitrary property someone left behind :)
Also the infinispan-xsite.xml sample doc does not use it either

Reply:
created:https://issues.redhat.com/browse/ISPN-11278